pkg/controller/v1alpha2/trial/trial_controller.go

```diff
@@ -109,6 +114,8 @@ var _ reconcile.Reconciler = &ReconcileTrial{}
 type ReconcileTrial struct {
 	client.Client
 	scheme *runtime.Scheme
+
+	managerclient.ManagerClient
```

Comment:
Will these be changes made for experiment controller too?

Reply:
Yeah, I will implement an interface in experiment, too. This will be in another PR since this PR is mainly focus on trial controller's test.

Reply:
SGTM